ansible/openshift-playbook.yml

```diff
@@ -73,42 +45,108 @@
       include_tasks: tasks/setup-ci-environ.yml
       vars:
         __test_harness_environ: staging
+        __test_harness_imgstream_name: linux-system-roles
         __test_harness_cm_name: config-staging
         __test_harness_cm_file: config-staging.json
-        __test_harness_ci_obj_file: ../openshift-objects-staging.yml
         __test_harness_dc: linux-system-roles-staging
         __test_harness_bc: linux-system-roles-staging
+        __test_harness_env_is_enabled: true
```

Comment:
How is this parameter used? (I could not find a place where this param was evaluated in this pr. Same for __test_harness_ghtoken_file.)

Reply:
`__test_harness_env_is_enabled` - hmm - it isn't used anymore
`__test_harness_ghtoken_file` is used in `ansible/tasks/create-secrets.yml`

Reply:
Ahh, sorry, I missed __test_harness_ghtoken_file. Yes, it is!

Reply:
removed `__test_harness_env_is_enabled` in https://github.com/linux-system-roles/test-harness/pull/95/commits/5238051d32d4de4232b63fe441a6ab2681a747f7